schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
-
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE SCHEMA `vet_clinic` ;
+CREATE TABLE `vet_clinic`.`animals` (
```

Comment:
- [ ] Kindly make the following corrections here:
- There is no need to create a schema called _**vet_clinic**_. The schema.sql creates or changes the table structures in PostgreSQL. 
- Secondly, the syntax for creating the _**animals**_ table is wrong. The correct syntax is `CREATE TABLE animals`.
- Kindly remove all backticks used in creating columns for the table. You can create a column for the table by doing this 
`name VARCHAR(45) NOT NULL,`. Kindly make these changes. 👍